### PR TITLE
Issue 125 update the text

### LINKED ISSUE #125: Update the Text on 3 of the pages
Hannah asked that some text on the pages be updated.  There are 3 total changes.  I attached her markups here as 3 images.  I am assigning this to Courtney to work on.

----------------
**1.**
IMG_4788

----------------
**2.**
IMG_4790

-----------------
**3.**
IMG_4791

### PR DESCRIPTION
**Pull request by cej8**
## Description

- Updated .yml files in Google Firebase for each game to make text no longer about the elections
- Updated text before each quiz
- Added override func stepViewControllerWillAppear(_ stepViewController: ORKStepViewController); which changes button from "Get Started" to "Chat Now" for the Chatroom Instruction step

## Referring Issues

- #125 
- 
- 

## Major changes

- 
- 
-

### CODE REVIEW

File: Jigsaw-app/Jigsaw/Info.plist [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>210210</string>
```
Comment (yo1995) [inline Jigsaw-app/Jigsaw/Info.plist]: We may revert this change and get rid of the build phase that updates the project version for now.

File: Jigsaw-app/Jigsaw/Controllers/Home/GameViewController.swift [hunk 2]
```diff
@@ -60,6 +60,13 @@ class GameViewController: ORKTaskViewController {
         return step
     }()
     
+    /// Changes continueButtonTitle from "Get Started" to "Chat Now" for the ChatroomInstruction step
+    override func stepViewControllerWillAppear(_ stepViewController: ORKStepViewController) {
+        if self.currentStepViewController?.step?.identifier == "ChatroomInstruction" {
```
Comment (yo1995) [inline Jigsaw-app/Jigsaw/Controllers/Home/GameViewController.swift]: Consider making the change on the following lines in `RoomProgressViewController` instead of here.

https://github.com/DukeMobileDevCenter/Jigsaw/blob/f3646ad9808da30e3215a5da8250c1f3a1548454/Jigsaw-app/Jigsaw/Controllers/Home/RoomProgressViewController.swift#L407

---

There are some cleanup to do with `RoomProgressViewController` though. I was hoping to separate it into at least 3 files

1. A file that handles all the UI updates on the Room Progress page, where we see all the doors, confetti balls and level information
2. A file to handle the communication with Firebase
3. A file to do the state management for `ORKTaskViewControllerDelegate`, as well as to fix the timing bug on wait page, #116

**Review by yo1995 — COMMENTED** [review]
Looking good, with only a suggestion on moving the `stepViewControllerWillAppear` delegate method to the other file.

**Comment by cej8** [comment]
> Looking good, with only a suggestion on moving the `stepViewControllerWillAppear` delegate method to the other file.

which file?

**Comment by yo1995** [comment]
> which file?

`GameViewController.swift`. Please see my comments above: https://github.com/DukeMobileDevCenter/Jigsaw/pull/131#discussion_r575934298
